src/Components/Web.JS/src/InputFile.ts

```diff
@@ -101,9 +101,9 @@ async function ensureArrayBufferReadyForSharedMemoryInterop(elem: InputElement,
   getFileById(elem, fileId).arrayBuffer = arrayBuffer;
 }
 
-async function readFileData(elem: InputElement, fileId: number, startOffset: number, count: number): Promise<string> {
+async function readFileData(elem: InputElement, fileId: number, startOffset: number, count: number): Promise<Uint8Array> {
   const arrayBuffer = await getArrayBufferFromFileAsync(elem, fileId);
-  return btoa(String.fromCharCode.apply(null, new Uint8Array(arrayBuffer, startOffset, count) as unknown as number[]));
+  return new Uint8Array(arrayBuffer, startOffset, count);
 }
```

Comment:
Required change to our current `InputFile` implementation as [we're expecting a `byte[]` return value](https://github.com/dotnet/aspnetcore/blob/d76ed601cab885211a73e8a90d16fd8b3f301736/src/Components/Web/src/Forms/InputFile/RemoteBrowserFileStream.cs#L60-L66).